### DISCUSSION TITLE
Passing Vite arguments to sanity build command

### DISCUSSION THREAD

**Discussion by lobunto** [sanity-io/sanity#3640] [Sanity Studio Support]
Since Sanity V3 are using Vite, I would suppose that I could pass [Vite] arguments to the `sanity build` command.
I've tried adding `--mode` like this `sanity build --mode staging`, but the mode (`import.meta.env.MODE`) always becomes `production`.

Has anyone else had this issue, or is it possible at all to pass or override Vite arguments?

**Comment by kmelve** [comment]
Hi! We're not doing a plain `vite build` so this isn't currently possible. However, you should be able to pass environment variables to `sanity.cli.ts`. 

I also want to ask how you planned to use `--mode staging`?
